Doc/tutorial/datastructures.rst

```diff
@@ -509,27 +509,24 @@ pair with ``del``. If you store using a key that is already in use, the old
 value associated with that key is forgotten.  It is an error to extract a value
 using a non-existent key.
 
-Performing ``list(d.keys())`` on a dictionary returns a list of all the keys
-used in the dictionary, in arbitrary order (if you want it sorted, just use
-``sorted(d.keys())`` instead). [2]_  To check whether a single key is in the
-dictionary, use the :keyword:`in` keyword.
+Performing ``list(d)`` on a dictionary returns a list of all the keys
+used in the dictionary, in insertion order. [2]_  To check whether a single
+key is in the dictionary, use the :keyword:`in` keyword.
```

Comment:
The footnote expains what a dict view is, but here we are making a list.

Reply:
I am sorry, which footnote do you mean?

Reply:
The `[2]_` markup is a footnote reference.